public/js/hotkey_binder/defaultKeys.json

```diff
@@ -0,0 +1,30 @@
+/**Add more elements here, along with a valid value for key
+* Elements keys must have the same name as their ID
+**/
+
+defaultKeys = {
+  "New Project": "Shift + N",
+  "Save Online": "Ctrl + S",
+  "Save Offline": "Ctrl + Alt + S",
+  "Download as Image": "Ctrl + D",
+  "Open Offline": "Ctrl + O",
+  "New Circuit": "Ctrl + Alt + N",
+  "Create Sub-circuit": "Shift + C",
+  "Combinational Analysis": "Shift + A",
+  "Start Plot": "Ctrl + P",
+  "Direction Up": "Up",
+  "Direction Down": "Down",
+  "Direction Left": "Left",
+  "Direction Right": "Right",
+  "Insert Label": "Ctrl + L",
+  "Label Direction Up": "Alt + Up",
+  "Label Direction Down": "Alt + Down",
+  "Label Direction Left": "Alt + Left",
+  "Label Direction Right": "Alt + Right",
+  "Move Element Up": "W",
+  "Move Element Down": "S",
+  "Move Element Left": "A",
+  "Move Element Right": "D",
+  "Hotkey Preference": "Ctrl + .",
```

Comment:
Ctrl + '/' is a better hotkey for preferences as Google Docs uses that.

Reply:
Unfortunately the key '/' came under the unsupported keys, hence I had added  "Ctrl + ." for it temporarily, looking for a suitable alternative.